src/views/VisitsView.vue

```diff
@@ -439,51 +440,58 @@ export default {
         this.canLoadMore = false;
         return;
       }
-      const eventsByDevice = result.rows;
-      if (mergeResults) {
-        this.count += result.numVisits;
-      } else {
-        this.count = result.numVisits;
-      }
       this.offset = result.queryOffset;
       this.canLoadMore = result.hasMoreVisits;
+
+
+      const summary = result.summary;
+      const filtered = result.visits.filter(
+        (visit: Visit): boolean =>
+          visit.events.length > 0 &&
+          visit.what != DefaultLabels.allLabels.bird.value &&
+          visit.what != DefaultLabels.allLabels.falsePositive.value
+      );
+      this.count += filtered.length;
```

Comment:
Seems like count (being only used for countMessage, which is used in the template), and which is dependent on the length of arrays, should really be a computed property, rather than book-kept in with all the other update logic.